pages/about.tsx

```diff
@@ -8,14 +9,12 @@ export default function About() {
         <title>About - My COVID Story</title>
         <link rel="icon" href="/favicon.ico" />
       </Head>
-      <Box w="95%" p={8} pb={24}>
-        <Box pb={4}>
-          <Heading as="h1" size="2xl" pb={4}>
-            About
-          </Heading>
-        </Box>
-        <Box pb={7}>
-          <Heading as="h2" size="l" pb={4}>
+      <ContentBox py>
```

Comment:
Main change here: adopted `ContentBox` for standard page width and responsive padding. I also decreased the padding between sections below to better balance with the padding.